modules/custom/social_lazy_loading/social_lazy_loading_images/src/SocialLazyLoadingImageDisplayOverride.php

```diff
@@ -82,40 +64,15 @@ public function loadOverrides($names) {
     ];
 
     foreach ($config_fields as $config_name => $field_name) {
-      $this->addImageOverride($config_name, $field_name, $overrides);
+      if (in_array($config_name, $names)) {
```

Comment:
@ronaldtebrake I've added this extra check as well. So we only override the right config 🙈 Please verify if this is implemented correctly 🤞